scripts/internal/manylinux-build-wheels.sh

```diff
@@ -38,7 +38,8 @@ for PYBIN in "${PYBINARIES[@]}"; do
     # Install dependencies
     ${PYBIN}/pip install --upgrade -r /work/requirements-dev.txt
 
-    build_type=MinSizeRel
+    build_type=""
+    compile_flags="-O2 -DNDEBUG"
```

Comment:
Just curious, why not also "-march=native -mtune=native" for the binaries that will only run on the exact same environment as the compiler build?

Not a big deal, but it seems that the tests may run faster.

Reply:
@hjmjohnson good idea. -march=native may cause issues, but -mtune=native is also already added -- thank you! -- here:

  https://github.com/InsightSoftwareConsortium/ITK/blob/cd38ccce01acf73b227e8b7a890dab8990ba2a9f/CMake/ITKSetStandardCompilerFlags.cmake#L244